### ISSUE TITLE
pip install vlite[ocr] -> ModuleNotFoundError: No module named 'onnxruntime'

### ISSUE THREAD

**Issue by jpetrantoni** [sdan/vlite#48]
```
(.venv) jpetrantoni@m1-mbp ~/a/workspace  uv pip install vlite[ocr]                                                                          (base) 
Resolved 49 packages in 525ms
   Built llama-cpp-python==0.2.62                                                                                                                   Downloaded 6 packages in 16.85s
Installed 49 packages in 482ms
 + annotated-types==0.6.0
 + beautifulsoup4==4.12.3
 + certifi==2024.2.2
 + charset-normalizer==3.3.2
 + diskcache==5.6.3
 + docx2txt==0.8
 + filelock==3.13.4
 + filetype==1.2.0
 + fsspec==2024.3.1
 + ftfy==6.2.0
 + huggingface-hub==0.22.2
 + idna==3.7
 + jinja2==3.1.3
 + llama-cpp-python==0.2.62
 + markupsafe==2.1.5
 + mpmath==1.3.0
 + networkx==3.3
 + numpy==1.26.4
 + opencv-python==4.9.0.80
 + packaging==24.0
 + pandas==2.2.2
 + pillow==10.3.0
 + pydantic==2.7.0
 + pydantic-core==2.18.1
 + pydantic-settings==2.2.1
 + pypdf2==3.0.1
 + pypdfium2==4.29.0
 + python-dateutil==2.9.0.post0
 + python-dotenv==1.0.1
 + pytz==2024.1
 + pyyaml==6.0.1
 + regex==2024.4.16
 + requests==2.31.0
 + safetensors==0.4.3
 + six==1.16.0
 + soupsieve==2.5
 + surya-ocr-vlite==0.3.0
 + sympy==1.12
 + tabulate==0.9.0
 + tiktoken==0.6.0
 + tokenizers==0.15.2
 + torch==2.2.2
 + tqdm==4.66.2
 + transformers==4.36.2
 + typing-extensions==4.11.0
 + tzdata==2024.1
 + urllib3==2.2.1
 + vlite==0.2.1
 + wcwidth==0.2.13
(.venv) jpetrantoni@m1-mbp ~/a/workspace  python vlite_test.py                                                                               (base) 
Traceback (most recent call last):
  File "/Users/joepetrantoni/ai/workspace/vlite_test.py", line 1, in <module>
    from vlite import VLite
  File "/Users/joepetrantoni/ai/workspace/.venv/lib/python3.10/site-packages/vlite/__init__.py", line 1, in <module>
    from .main import VLite
  File "/Users/joepetrantoni/ai/workspace/.venv/lib/python3.10/site-packages/vlite/main.py", line 3, in <module>
    from .model import EmbeddingModel
  File "/Users/joepetrantoni/ai/workspace/.venv/lib/python3.10/site-packages/vlite/model.py", line 3, in <module>
    import onnxruntime as ort
ModuleNotFoundError: No module named 'onnxruntime'
```

**Comment by jpetrantoni** [comment]
pip install vlite does not cause this issue

**Comment by sdan** [comment]
fixed in v0.2.4 

try pip install vlite[ocr]==0.2.4
